pkg/shell/machine-dialogs.js

```diff
@@ -417,6 +417,9 @@ define([
 
             dialog.run(try_to_connect(dialog.address), function (ex) {
                 if (ex.problem == "no-host") {
+                    if (dialog.address.indexOf(":") == -1)
+                        dialog.address += ":22";
+                    ex.message = "Cockpit could not contact the given host "+dialog.address+". Make sure it has ssh running on port 22, or specify another port in the address.";
```

Comment:
`` ex.message = cockpit.format(_("Cockpit could not contact the given host $0. Make sure it has ssh running on port 22, or specify another port in the address."),dialog.address);``  
 How to check string translation in cockpit? When I tried changing the Display language, it reloads to same (english)?